src/Modules/Backend/assets/Backend/webpack/scss/components/_data-grids.scss

```diff
@@ -16,7 +15,7 @@
     max-width: 70%;
   }
 
-  .sortHandle {
+  .sort-handle {
```

Comment:
@carakas I have a task for @daphneslootmans to cleanup CSS and JS when fork6 is a bit further. 
Let's focus here on the CI issue, otherwise this PR will get out of focus.